ClingoFluent command
ClingoFluent sense_fluent
ClingoFluent[] evaluate_fluents
---
bool success
string status
ClingoFluent[] observable_fluents
float32 plan_cost
---
string status
ClingoFluent[] observable_fluents
